todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.java

```diff
@@ -94,23 +163,20 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         fab.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                mPresenter.editTask();
+                editTask();
             }
         });
 
-        return root;
-    }
+        mViewModel = Injection.provideTaskDetailsViewModel(getTaskId(), getContext());
```

Comment:
put this somewhere more visible for example within bind(). its too confusing like this.